
initial_pose_pub:
  ros__parameters:
    initial_pose:
      x: 0.0
      y: 0.0
    old_trajectory_id: 0  # Cartographer old trajectory ID to be deleted initially
    cycle_period: 2.0     # cycling period for the state machine
